Xwt.Gtk/Xwt.GtkBackend.CellViews/CustomCellRenderer.cs

```diff
@@ -57,49 +58,62 @@ internal void StartDrawing (Rectangle ba, Rectangle ca, Gtk.CellRendererState fl
 			hasFocus = (flags & CellRendererState.Focused) != 0;
 			isPrelit = (flags & CellRendererState.Prelit) != 0;
 
-			isDrawing = true;
+			// Gtk will rerender the cell on every status change, hence we can expect the values
+			// set here to be valid until the geometry or any other status of the cell and tree changes.
+			// Setting shown=true ensures that those values are always reused instead if queried
+			// from the parent tree after the cell has been rendered.
```

Comment:
Let's say you have a tree with 10 rows. After rendering the 10 rows, the data cached in the renderer is from the 10th row. Now if I move the mouse over the first row, it will be using the cellArea from the 10th row. Moving the mouse over a cell doesn't change its status, and the tree has no need to render it.

Reply:
That's the point, it WILL redraw, I don't know the exact logic responsible for this, but even if you disable all the events the TreeView will invalidate an area behind the mouse pointer and redraw the renderes inside that area.

Reply:
Actually it will call the data func, which will Load the cell data and that will result in a redraw.

Reply:
OK then.